src/zope/index/field/index.py

```diff
@@ -84,8 +76,9 @@ def index_doc(self, docid, value):
     def unindex_doc(self, docid):
         """See interface IInjection"""
         rev_index = self._rev_index
-        value = rev_index.get(docid)
```

Comment:
I'm not familiar enough with python interning rules to know how to do this in a completely safe way.  Is there a part of the python spec that discusses this or is it something that can be different from implementation to implementation.